rest/taskana-rest-spring/src/test/java/pro/taskana/rest/TestSchemaNameCustomizable.java

```diff
@@ -57,7 +58,7 @@ void checkCustomSchemaNameIsDefined_Postgres() throws SQLException {
           String tableName = rs.getString("tablename");
           tablefound = tableName.equals("workbasket");
         }
-        Assertions.assertTrue(tablefound, "Table workbasket should be there ...");
+        assertThat(tablefound).as("Table workbasket should be there ...").isTrue();
```

Comment:
as is wrong. (As far as I understand it.) Please use isEqualTo()

Reply:
tablefound is a boolean, that's why I don't understand why I should use isEqualTo(). As far as I can see, we obtain the same results with the old and the new expression... am I wrong? the String is shown in case the boolean is false.

Reply:
nevermind.. I was wrong. Thanks!